torch/csrc/jit/script/compiler.cpp

```diff
@@ -1871,11 +1886,38 @@ struct to_ir {
           {rhs},
           {},
           self);
-
+    } else if (lhsValue->type()->kind() == TypeKind::ClassType) {
+      // Call `__iadd__` so updates happen in place on class types
+      // https://docs.python.org/3/reference/datamodel.html#object.__iadd__
+      auto magic_method_name = getAugMagicMethod(stmt);
+      const auto rhs = NamedValue(stmt.rhs().range(), emitExpr(stmt.rhs()))
+                           .value(*method.graph());
```

Comment:
Why do we need to do `NamedValue(stmt.rhs().range(), emitExpr(stmt.rhs())).value(*method.graph())` instead of just `emitExpr` ?